release-tool

```diff
@@ -540,24 +515,28 @@ merge() {
         fi
     fi
 
-    CHANGELOG=$(grep -Ezo "## ${RELEASE_NAME} \([0-9]{4}-[0-9]{2}-[0-9]{2}\)\n\n(.|\n)+?\n\n## " CHANGELOG.md \
+    local flags="-Pzo"
+    if OS_MACOS; then
+        flags="-Ezo"
+    fi
+    CHANGELOG=$(grep ${flags} "## ${RELEASE_NAME} \([0-9]{4}-[0-9]{2}-[0-9]{2}\)\n\n(.|\n)+?\n\n## " CHANGELOG.md \
```

Comment:
I had to use -P on MSYS2 to make this work at all. On macOS it uses -E.

Reply:
The two syntaxes are not fully compatible, but I can see that GNU grep seems to be ignoring the `-z` somehow. Not sure what's going on there.